ci/linux-sgx-centos8-docker-gcc-release.jenkinsfile

```diff
@@ -46,8 +46,8 @@ node (node_label) {
                 load '.ci/lib/config-release.jenkinsfile'
 
                 load '../ci/stage-build-sgx.jenkinsfile'
-                load '../ci/stage-test.jenkinsfile'
                 load '../ci/stage-test-sgx.jenkinsfile'
+                load '../ci/stage-test.jenkinsfile'
```

Comment:
what is this change for? generally unrelated refactoring should be done in a separate PR.

Reply:
stage-test.jenkinsfile has workload_parser in the end which parses all the workload results in the end. Since CI examples has separate file, that's why this code was needed at the end. So to generalize this piece of code is added at end of jenkinsfile